designing-reports/report-designer-tools/web-report-designer/how-to-set-up-in-blazor-application.md

```diff
@@ -27,7 +27,7 @@ previous_url: /how-to-blazor-web-report-designer
 
 1. Add required settings in the Startup.cs file.The __ConfigureServices__ method inside the `Startup.cs` in the project should be modified in order to enable the Web Report Designer REST service. Make sure the application is configured for WebAPI controllers and call the *AddNewtonsoftJson* to enable the required NewtonsoftJson serialization: 
     
-	````C#
+````C#
```

Comment:
Removing the spaces in front of the backticks breaks the ordered list:
![image](https://user-images.githubusercontent.com/60105689/191911266-6ba1c7d4-f637-47f9-ae8e-6531214fcd15.png)

To render the list correctly, use three spaces before the opening and closing backticks and make sure that none of the lines that belong to the code block starts with less than three spaces:

![image](https://user-images.githubusercontent.com/60105689/191912216-1282eced-6597-4f68-b220-00635c5cfe77.png)